docs/SECURITY.md

```diff
@@ -187,7 +187,113 @@ If users want to setup 2-way auth
 9. logviewer.https.want.client.auth (If this set to true server requests for client certificate authentication, but keeps the connection if no authentication provided)
 10. logviewer.https.need.client.auth (If this set to true server requires client to provide authentication)
 
+## Mutual TLS (mTLS) Support
 
+**Available since Storm 2.7.0 (STORM-4070)**
+
+Storm now supports mutual TLS for its internal Thrift RPC between Nimbus, Supervisors, and workers. mTLS goes beyond one-way TLS by requiring **both** sides to present and verify certificates, giving full two-way certificate authentication *and* encryption.
```

Comment:
Maybe something like: "Storm now supports mutual TLS (mTLS) for internal Thrift RPC communication among Nimbus, Supervisors, and workers. Unlike one-way TLS, mTLS requires both parties to present and verify each other's certificates. This ensures full two-way certificate authentication and encryption." 

(introduces mTLS as abbreviation first)